zipline/finance/performance/period.py

```diff
@@ -550,8 +548,8 @@ def as_account(self):
             getattr(self, 'day_trades_remaining', float('inf'))
         account.leverage = getattr(self, 'leverage',
                                    period_stats.gross_leverage)
-        account.net_leverage = period_stats.net_leverage
-
+        account.net_leverage = getattr(self, 'net_leverage',
```

Comment:
I'm not sure I follow, but previously we always calculated `net_leverage` from the positions zipline records in the position tracker. This change allows us to get a value from the broker, just like the other account attributes